src/libecalc/presentation/yaml/yaml_types/models/yaml_compressor_trains.py

```diff
@@ -62,6 +62,11 @@ class YamlSingleSpeedCompressorTrain(YamlCompressorTrainBase):
         description="Constant to adjust power usage in MW",
         title="POWER_ADJUSTMENT_CONSTANT",
     )
+    power_adjustment_factor: float = Field(
```

Comment:
inform in docs about this new feature, and when  (8.22) it was introduced in keyword section for POWER_ADJUSTMENT_FACTOR (and _CONSTANT?)